ynput/ayon-core
# Chore: Validate Version plugin enabled based on profiles

## Changelog Description
Plugin `ValidateVersion` is enabled, optional and active based on profiles in settings.

## Additional info
This is one of many PRs that tries to resolve plugins in core that have host name filtering.

## Testing notes:
1. Create core package.
2. Upload the package to server.
3. Make sure the AYON launcher is using new core client code.
4. Validate Version should work with default settings the same way as did before this PR.
5. Changing the settings should take effect in Publisher (e.g. change behavior for a host).
